### ISSUE TITLE
Plugin doesn't work

### ISSUE THREAD

**Issue by sashasashasasha151** [rikvdkleij/intellij-haskell#618]
I successfully builded project with `stack build --fast`, run `Update Settings and restart REPLs` and in messages it show `indigo-0.2.1: unregistering (missing dependencies: cleveland)` and plugin features don't work (beta83)

**Comment by sashasashasasha151** [comment]
Maybe you can help us more widely. We develop https://gitlab.com/morley-framework/morley/-/tree/master but looks like there are a lot of issues with this repo and haskell plugin and we can't fix them all. It a bit works with beta72 but not also too bad(

**Comment by rikvdkleij** [comment]
I have imported the project. 

Main issue is that your project has two `stack.yaml`s. That's not supported by plugin.
Btw, you can enable the Haskell Event log to see more info about what is going on.

**Comment by sashasashasasha151** [comment]
Okay, sorry, it was my bad, because I didn't update IDEA
Now I did it, build project, run `Update Settings and restart REPLs and this is logs after my click on the simple `let` element
https://gist.github.com/sashasashasasha151/bd92a91ce863bb3277d5031609418de8

Also about two `stack.yaml`s. I think you misunderstood smth because we have only one `stack.yaml` file

**Comment by rikvdkleij** [comment]
Yes, I saw the same errors. Looks like there is some conflict between the library packages. And then I saw the second stack.yaml...

For performance reason all library packages are loaded in one GHCi instance.

But yeah, it’s some guessing

**Comment by sashasashasasha151** [comment]
We also have mixins and custom prelude there, maybe it is also the problem

**Comment by rikvdkleij** [comment]
Mixins are not supported by Stack, see https://github.com/commercialhaskell/stack/issues/5077
